src/Compilers/CSharp/Test/Emit/Emit/AsyncMainEntryPointTests.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Emit
+{
+    public class AsyncMainEntryPointTests : CompilingTestBase
```

Comment:
>AsyncMainEntryPointTests [](start = 17, length = 24)

I don't understand why are we splitting test between multiple files. Please putt all tests into CodeGenAsyncMainTest.cs